docs/tutorials/otio-file-format-specification.md

```diff
@@ -27,7 +25,7 @@ OTIO does not support instancing, so you cannot reference the same object multip
 
 The top level object in an OTIO file can be any OTIO data type, but is typically a Timeline. This means that most use cases will assume that the top level
 object is a Timeline, but in specific workflows, you can read and write otio files that contain
-just a Clip, Sequence, RationalTime, or any other OTIO data type. Due to the nature of JSON, you could also read/write an array of objects, but we
+just a Clip, Track, RationalTime, or any other OTIO data type. Due to the nature of JSON, you could also read/write an array of objects, but we
 recommend that you use the OTIO SerializableCollection data type in this case so that you can attach metadata to the container itself. Code that reads an OTIO
```

Comment:
```suggestion
recommend using the OTIO SerializableCollection data type in this case so that metadata can be attached to the container itself. Code that reads an OTIO
```